docs/database/elasticsearch.md

```diff
@@ -14,12 +14,13 @@ described on this page.
 
 ## Span Name
 
-The **span name** SHOULD be of the format `<endpoint id>`.
+The **span name** SHOULD be of the format `{db.operation.name} {db.collection.name}`.
```

Comment:
I wonder if this section is necessary, it seems to be perfectly aligned with the general db span name guidance now.

Reply:
Yeah, that's a good point, but a bit tricky as well. In general what I propose here is very similar to the general db span name guideline, but it differs a bit, specifically: `db.collection.name` isn't available for all queries - there could be queries that target all indices or no index - in that case we won't have this value and it'd be very unfortunate to fall back to the next item in the `{target}` list from the general spec. I think the `target` list is very useful when all spans have the same set of attributes, but if the list of attributes changes within the same instrumentation (and potentially within the same trace), then we need a solution for that, otherwise the span name format will be mixed. 

I opened https://github.com/open-telemetry/semantic-conventions/issues/1045 to discuss this.

Reply:
I think https://github.com/open-telemetry/semantic-conventions/pull/1069 is going to get merged. Then this is addressed, no?

Reply:
Yes, this is addressed - I removed this part. 

Nevertheless I still have 1 sentence about the span name calling out specifically that we want to use the endpoint id and the index name, instead of the url path. I think that's useful info for instrumentation. But other than that, I just reference the general span name now.